core/iwasm/libraries/lib-socket/test/nslookup.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2019 Intel Corporation.  All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#include <assert.h>
+#include <string.h>
+#ifdef __wasi__
+#include <wasi_socket_ext.h>
+#endif
+#include <netdb.h>
+
+void
+testNsLookup(int af)
```

Comment:
in WAMR we use `sneak_casing` everywhere, so it'd be good to stick to that in tests too.